views/windows/featureedit.html

```diff
@@ -16,6 +16,17 @@
           <simplemde text="ctrl.description" edit="true" is-required="false" maxlength="4000"></simplemde>
         </div>
       </div>
+
+      <div ng-if="ctrl.customFoldersEnabled" class="d-flex flex-row form-group">
+        <label class="col-2 col-form-label text-right">Folder</label>
```

Comment:
This markup doesn't match the other rows in the form, and is misaligned.

![Screen Shot 2019-05-21 at 11 40 35 AM](https://user-images.githubusercontent.com/7914731/58118053-4b11bd80-7bbd-11e9-9e87-d31b430727db.png)

Reply:
Fixed